libcontainer/intelrdt/intelrdt.go

```diff
@@ -563,6 +562,14 @@ func (m *intelRdtManager) Apply(pid int) (err error) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 
+	if m.config.IntelRdt.ClosID != "" && m.config.IntelRdt.L3CacheSchema == "" && m.config.IntelRdt.MemBwSchema == "" {
+		// Check that the CLOS exists, i.e. it has been pre-configured to
+		// conform with the runtime spec
+		if _, err := os.Stat(path); err != nil {
+			return fmt.Errorf("unable to access %q (must be pre-created when l3CacheSchema and memBwSchema are empty): %v", path, err)
```

Comment:
1. an error from os.Stat already contains path, no need to have it twice in the error message.
2. use `%w` for error.

Reply:
Valid point, changed now to `"clos dir not accessible (must be pre-created when l3CacheSchema and memBwSchema are empty): %w"`